src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Infrastructure/IAanHubRestApiClient.cs

```diff
@@ -95,6 +96,10 @@ public interface IAanHubRestApiClient
         [Body] JsonPatchDocument<PatchMemberRequest> patchMemberRequest,
         CancellationToken cancellationToken);
 
+    [Post("members/{memberId}/Leaving")]
+    [AllowAnyStatusCode]
+    Task<Response<PostMemberLeavingResponse>> PostMembersLeaving([Path] Guid memberId, [Body] PostMemberLeavingModel model, CancellationToken cancellationToken);
```

Comment:
Because you do not have a response body as such, use `Task<HttpResponseMessage>`  instead, this will allow you to get rid of empty response type as well.

Reply:
That's good, will do